library/std/src/os/unix/net/datagram.rs

```diff
@@ -1,24 +1,7 @@
-#[cfg(any(
-    doc,
-    target_os = "android",
-    target_os = "dragonfly",
-    target_os = "emscripten",
```

Comment:
this seems to have removed emscripten as well which isn't mentioned in the PR, was this intentional?

Reply:
> this seems to have removed emscripten as well which isn't mentioned in the PR, was this intentional?

Comment not resolved too, yes?

Reply:
I merged this intentionally knowing that it did remove other platforms as well.